### PR TITLE
requested updates:

### PR DESCRIPTION
**Pull request by jeffrummel**
- sets fixed height with view more toggle on bios
- adds title to Michael Berry

### CODE REVIEW

File: layouts/_default/board-of-directors-page.html [hunk 1]
```diff
@@ -67,8 +67,49 @@ <h2 class="mb-5 text-5xl font-black">{{ . }}</h2>
...
+            {{ partial "helper/svg.html" "plus" }}
+          </svg>
+        </span>
+        <span class="ml-2 align-bottom uppercase hover:underline">
+          Show more…
+        </span>
+      </button>
+      <button
+        x-show="isOpen"
+        :aria-expanded="''+isOpen"
```
Comment (jeffrummel) [inline layouts/_default/board-of-directors-page.html:97]: @carlmjohnson This button closes the bio to show less. What's the correct aria label here?

**Comment by carlmjohnson** [comment]
image

Her bio is too short to need show more, but it has the button anyway. Maybe just make the boxes smaller so that she needs a show more button too? Every once in a while we'll have a feature with a very short sidebar that has this issue too, but it's not so common.

File: layouts/_default/board-of-directors-page.html (hunk 1 above)
Comment (carlmjohnson) [reply layouts/_default/board-of-directors-page.html:97]: You could add an ID to the "Show less…" span and then do `aria-label="{{ $showLessID }}"` here.

Comment (carlmjohnson) [reply layouts/_default/board-of-directors-page.html:97]: Does it need an aria-label? It might just be okay without it, since that's the content anyway…

Comment (jeffrummel) [reply layouts/_default/board-of-directors-page.html]: These aria labels are removed

**Pull request merged**